addons/tagging/stringtable.xml

```diff
@@ -26,7 +26,7 @@
             <Chinese>定義噴漆系統預設設定</Chinese>
         </Key>
         <Key ID="STR_ACE_Tagging_QuickTag">
-            <English>Quick Tag</English>
+            <English>Spray Paint - Quick Tag</English>
```

Comment:
Remove other translations?

Reply:
I just wanted to clarify what the keybind was for
I don't think we should drop the other languages because they might not be updated in time for next release